### PR TITLE
[PLANG-138] Add possibility to mark envs as sensitive and retrieve only sensitives

### PR DESCRIPTION
**Pull request by trapacska**
# Usage

## To mark an env sensitive
`envman add --key KEY --value value --sensitive`

## To list only sensitive
`envman print --json --only-sensitives`

### CODE REVIEW

File: cli/print.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func printRawEnvs(envList models.EnvsJSONListModel) {
 	fmt.Println()
 }
 
-func convertToEnsJSONModel(envs []models.EnvironmentItemModel, expand bool) (models.EnvsJSONListModel, error) {
+func convertToEnsJSONModel(envs []models.EnvironmentItemModel, expand, sensitiveOnly bool) (models.EnvsJSONListModel, error) {
```
Comment (imrekel) [inline cli/print.go:32]:
```suggestion
func convertToEnvsJSONModel(envs []models.EnvironmentItemModel, expand, sensitiveOnly bool) (models.EnvsJSONListModel, error) {
```

File: cli/print.go [hunk 2]
```diff
@@ -65,14 +71,15 @@ func print(c *cli.Context) error {
...
+	sensitiveOnly := c.Bool(SensitiveOnlyKey)
 
 	// Read envs
 	environments, err := envman.ReadEnvs(envman.CurrentEnvStoreFilePath)
 	if err != nil {
 		log.Fatalf("Failed to read envs, error: %s", err)
 	}
 
-	envsJSONList, err := convertToEnsJSONModel(environments, expand)
+	envsJSONList, err := convertToEnsJSONModel(environments, expand, sensitiveOnly)
```
Comment (imrekel) [inline cli/print.go:82]:
```suggestion
	envsJSONList, err := convertToEnvsJSONModel(environments, expand, sensitiveOnly)
```

File: cli/print_test.go [hunk 3]
```diff
@@ -18,15 +18,39 @@ envs:
 	environments, err := envman.ParseEnvsYML([]byte(envsStr))
 	require.Equal(t, nil, err)
 
-	envsJSONList, err := convertToEnsJSONModel(environments, false)
+	envsJSONList, err := convertToEnsJSONModel(environments, false, false)
```
Comment (imrekel) [inline cli/print_test.go]:
```suggestion
	envsJSONList, err := convertToEnvsJSONModel(environments, false, false)
```

File: cli/print_test.go [hunk 4]
```diff
@@ -18,15 +18,39 @@ envs:
...
 	require.Equal(t, nil, err)
 	require.Equal(t, "$HOME", envsJSONList["TEST_HOME1"])
 	require.Equal(t, "$TEST_HOME1/test", envsJSONList["TEST_HOME2"])
 
 	testHome1 := os.Getenv("HOME")
 	testHome2 := path.Join(testHome1, "test")
-	envsJSONList, err = convertToEnsJSONModel(environments, true)
+	envsJSONList, err = convertToEnsJSONModel(environments, true, false)
```
Comment (imrekel) [inline cli/print_test.go]:
```suggestion
	envsJSONList, err = convertToEnvsJSONModel(environments, true, false)
```

**Review by imrekel — APPROVED** [review]
LGTM

**Review by ofalvai — APPROVED** [review]

**Comment by ofalvai** [comment]
Changes look good. (Envman doesn't use go.mod yet, I'll open a PR about this soon)

**Review by godrei — APPROVED** [review]

**Pull request merged**